doc/nrf/ug_nrf5340.rst

```diff
@@ -106,165 +85,246 @@ Zephyr includes the `OpenAMP`_ library, which provides a complete solution for e
-Name the file *network_sample*\ .conf, where *network_sample* is the name of the child image (for example, ``hci_rpmsg.conf``).
-Place the file in a :file:`child_image` subfolder of the application sample directory.
-This allows alignment of the configurations of both images.
-For example, see the :ref:`ble_throughput` child image configuration in :file:`nrf/samples/bluetooth/throughput/child_image/hci_rpmsg.conf`.
 
-Application samples
-===================
+Direct use of the radio peripheral
+==================================
 
-The |NCS| provides a series of :ref:`Bluetooth Low Energy samples <ble_samples>`, in addition to the :ref:`Bluetooth samples in Zephyr <zephyr:bluetooth-samples>`.
-Most of these samples should run on the nRF5340 DK, but not all have been thoroughly tested.
+.. list-table::
+   :header-rows: 1
 
-Some Bluetooth LE samples require configuration adjustments to the :ref:`zephyr:bluetooth-hci-rpmsg-sample` sample as described in the `Network samples`_ section.
+   * - Network core
+     - Application core
+   * - | :ref:`direct_test_mode`
+       | :ref:`radio_test`
+       | :ref:`timeslot_sample`
+     - :ref:`nrf5340_empty_app_core`
 
-The |NCS| also provides :ref:`Thread <ug_thread>`, :ref:`Zigbee <ug_zigbee>`, and :ref:`Matter <ug_matter>` samples that can be run on the nRF5340 DK.
-These samples are built for the application core, but make use of both cores.
-The network core application is built and programmed automatically.
+.. note::
+   The above list might not be exhaustive.
 
-Additionally, the |NCS| provides :ref:`NFC samples <nfc_samples>` that are available for nRF5340.
-These samples run only on the application core and do not require any firmware for the network core.
+Samples that directly use the radio peripheral can run on the network core of the nRF5340.
+They do not require any functionality from the application core.
 
-When programming any of these samples to the application core, configure :option:`CONFIG_BOARD_ENABLE_CPUNET` to select whether the network core should be enabled.
-When radio protocols (Bluetooth LE, IEEE 802.15.4) are used, this option is enabled by default.
+However, on nRF5340, the application core is responsible for starting the network core and connecting its GPIO pins (see :option:`CONFIG_BOARD_ENABLE_CPUNET` and the code in :file:`zephyr/boards/arm/nrf5340dk_nrf5340/nrf5340_cpunet_reset.c`).
 
-.. _ug_nrf5340_building:
+Therefore, the application core must always be programmed, even if the firmware is supposed to run only on the network core.
+You can use the :ref:`nrf5340_empty_app_core` sample for this purpose.
+This sample does not automatically include the network core application as child image, so you must either configure it appropriately or program the images for the network core and application core separately.
```

Comment:
Note that the `empty_app_core` sample will be build automatically for a network core sample which sets the correct option. This is the recommended way of working with a "network core only" sample. 

![image](https://user-images.githubusercontent.com/1547348/121817114-9d7c1600-cc7f-11eb-89d3-bcc62e642a63.png)

Reply:
Correct, netcore fw is parent image.